test/test_transforms.py

```diff
@@ -1314,6 +1314,25 @@ def test_gaussian_blur_asserts():
         transforms.GaussianBlur(3, "sigma_string")
 
 
+def test_gaussian_noise():
+    np_img = np.ones((100, 100, 3), dtype=np.uint8) * 255
+    img = F.to_pil_image(np_img, "RGB")
+    out = transforms.GaussianNoise(2.0, (0.1, 2.0))(img)
+    assert isinstance(out, PIL.Image.Image)
```

Comment:
Probably tests are failing due to this. 
Also can you plz merge the latest main branch? That will avoid a few CI errors

Reply:
Yes will do thanks!